### PR TITLE
docs: changelog 5.10.1

### PR DESCRIPTION
**Pull request by MadCcc**
- ⚡️ Optimize CSS-in-JS Design Token cache matching. [#45302](https://github.com/ant-design/ant-design/pull/45302)
- 🆕 Checkbox.Group &amp; Radio.Group `options` add missing `id` props. [#45287](https://github.com/ant-design/ant-design/pull/45287)
- 🐞 Fix Watermark that can be hidden via "Hide Element" feature in browser. [#45290](https://github.com/ant-design/ant-design/pull/45290) [@Yuiai01](https://github.com/Yuiai01)
- 🐞 Fix Input that background should not be transparent when hovered or focused. [#45297](https://github.com/ant-design/ant-design/pull/45297) [@MadCcc](https://github.com/MadCcc)
- 🐞 Fix Form call `resetFields` will still keep Form.List field when its `initialValue` is set. [#45284](https://github.com/ant-design/ant-design/pull/45284)
- 🐞 Fix DirectoryTree `selectedNodes` in `onSelect` method could not get a value when configuring `fieldNames`. [#45036](https://github.com/ant-design/ant-design/pull/45036) [@Zian502](https://github.com/Zian502)
- 💄 Revert outline style of Input, InputNumber, Select, Cascader, TreeSelect, DatePicker, TimePicker, ColorPicker. [#45286](https://github.com/ant-design/ant-design/pull/45286) [@MadCcc](https://github.com/MadCcc)
- 💄 Fix Card style with small size Tabs. [#45272](https://github.com/ant-design/ant-design/pull/45272) [@MadCcc](https://github.com/MadCcc)

---

- ⚡️ 优化 CSS-in-JS Design Token 缓存命中性能。[#45302](https://github.com/ant-design/ant-design/pull/45302)
- 🆕 为 Checkbox.Group 与 Radio.Group 的 `options` 添加 `id` 属性支持。[#45287](https://github.com/ant-design/ant-design/pull/45287)
- 🐞 修复 Watermark可以使用浏览器的 `Hide Element` 来隐藏水印。[#45290](https://github.com/ant-design/ant-design/pull/45290) [@Yuiai01](https://github.com/Yuiai01)
- 🐞 修复 Input 组件 hover 或者 focus 状态时背景色变为透明的问题。[#45297](https://github.com/ant-design/ant-design/pull/45297) [@MadCcc](https://github.com/MadCcc)
- 🐞 修复 Form 在调用 `resetFields` 时，无法重置配置了 `initialValue` 的 Form.List 的问题。[#45284](https://github.com/ant-design/ant-design/pull/45284)
- 🐞 修复 DirectoryTree 在配置 `fieldNames` 时，`onSelect` 方法中的 `selectedNodes` 无法获取值。[#45036](https://github.com/ant-design/ant-design/pull/45036) [@Zian502](https://github.com/Zian502)
- 💄 回滚 Input、InputNumber、Select、Cascader、TreeSelect、DatePicker、TimePicker、ColorPicker 的描边样式。[#45286](https://github.com/ant-design/ant-design/pull/45286) [@MadCcc](https://github.com/MadCcc)
- 💄 修复 Card 搭配小尺寸 Tabs 时的样式问题。[#45272](https://github.com/ant-design/ant-design/pull/45272) [@MadCcc](https://github.com/MadCcc)

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 7b62779</samp>

This pull request prepares antd for the 5.10.1 patch release. It adds the release notes in both English and Chinese to the `CHANGELOG` files, and updates the version number in the `package.json` file.

### <samp>🤖 Generated by Copilot at 7b62779</samp>

*  Update version number to 5.10.1 in `package.json` ([link](https://github.com/ant-design/ant-design/pull/45329/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L3-R3))
*  Add release notes for 5.10.1 in English and Chinese in `CHANGELOG.en-US.md` and `CHANGELOG.zh-CN.md` ([link](https://github.com/ant-design/ant-design/pull/45329/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cR19-R31), [link](https://github.com/ant-design/ant-design/pull/45329/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4R19-R31))

### CODE REVIEW

**Review by kiner-tang — APPROVED** [review]

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/45310

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/45333

**Comment by li-jia-nan** [comment]
- #45314

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
 
 ---
 
+## 5.10.1
+
+`2023-10-12`
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:21]:
```suggestion
`2023-10-14`
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
 
 ---
 
+## 5.10.1
+
+`2023-10-13`
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:21]:
```suggestion
`2023-10-14`
```

Comment (yoyo837) [reply CHANGELOG.zh-CN.md:21]: @li-jia-nan 直接改到明天, 今天大佬应该也不会发.

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md:21]: 过了12点再改，不着急，我昨天晚上已经改过一次了

**Pull request merged**